delfin/api/views/storages.py

```diff
@@ -30,3 +31,21 @@ def build_storage(storage):
     else:
         view['sync_status'] = 'SYNCING'
     return dict(view)
+
+
+def build_capabilities(storage_info, capabilities):
+    """build capability API response"""
+
+    # validate capabilities
+    validate_capabilities(capabilities)
```

Comment:
please move validate_capabilites(capabilities) to v1/storages.py. view/storages.py is in charge of building the response.

Reply:
@wisererik Comment addressed and test